src/util/system.cpp

```diff
@@ -350,11 +410,28 @@ Optional<unsigned int> ArgsManager::GetArgFlags(const std::string& name) const
     return nullopt;
 }
 
+bool ArgsManager::CheckArgFlags(const std::string& name,
+    unsigned int require,
+    unsigned int forbid,
+    const char* context) const
```

Comment:
In commit "Implement missing error checking for ArgsManager flags" (4b8149b5aae874cffe81423a2d4529c6f5b6e944)

Note: comment describing this function is in the header file shown below on github